types.ts

```diff
@@ -433,6 +433,20 @@ export interface OAuth2Authentication extends BaseAuthentication {
    * See https://datatracker.ietf.org/doc/html/rfc7636 for more details.
    */
   useProofKeyForCodeExchange?: boolean;
+
+  /**
```

Comment:
Thinking aloud... I wonder if it'd make sense at some point to have a way to clearly tag certain new options as "experimental" in the SDK so users know that those options are new and their behavior could change in future SDK releases.

If we had a way to mark experimental options, seems like an unusual corner case with only one known 1st-party-pack usage might be a good use case for it.

Reply:
Yeah, would be nice. Possibly we could use the `@internal` or `@category` options that are available in TypeDoc?

https://typedoc.org/guides/doccomments/

Reply:
I like this idea, but I feel like it deserves more thought beyond the urgent need of avoiding slack downtime, so I'd like to defer.